spec/cpu_choice_spec.rb

```diff
@@ -56,4 +56,35 @@
       expect(@cpu.cpu_first_pick(@playing_board)).to eq('9')
     end
   end
+
+  describe '.minimax' do
```

Comment:
This is actually a design hint. You're directly calling `minimax` in the tests even though `minimax` isn't in the public interface (it's public, but it's not meant to be used like this). POODR section 4.4 talks about this. 

OTOH if minimax was a separate unit, maybe a function or lambda or another class, you could inject it into the CPU and test it directly. That way you could swap a different algorithm in if you wanted for different computer players.

Reply:
Makes sense to me now. Should be easy to fix